src/System.IO.FileSystem/tests/Performance/Perf.Directory.cs

```diff
@@ -58,33 +58,54 @@ public void Exists()
             Directory.Delete(testFile);
         }
 
-        [Benchmark]
-        [InlineData(10, 100)]
-        [InlineData(100, 10)]
-        [InlineData(1000, 1)]
-        [OuterLoop("Takes a lot of time to finish")]
-        public void RecursiveCreateDirectory(int depth, int times)
+        public string GetTestDeepFilePath(int depth)
         {
-            if (!PlatformDetection.IsWindows && depth == 1000)
-                return;
-            
-            // Setup
-            string rootDirectory = GetTestFilePath();
-            StringBuilder sb = new StringBuilder(5000);
+            string directory = Path.DirectorySeparatorChar + "a";
+            StringBuilder sb = new StringBuilder(depth * 2);
             for (int i = 0; i < depth; i++)
             {
-                sb.Append(@"\a");
+                sb.Append(directory);
             }
 
-            string path = sb.ToString();
+            return sb.ToString();
+        }
+
+        public static TheoryData<int ,int> RecursiveDepthData
+        {
+            get
+            {
+                var data = new TheoryData<int, int>();
+                data.Add(10, 100);
+
+                // Length of the path can be atmost 260 characters on netfx.
```

Comment:
Atmost

Reply:
You aren’t using this on netfx though the tests are skipped?

Reply:
@danmosemsft i doubt ```[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]``` is working for performance tests as ```[PlatformSpecific(Testplatforms.unix)]``` was not working. thats y i added this to avoid the error incase its not working.
I am not able to run performance tests locally against netfx